### PR TITLE
Add Elijah Capps to Website

### PR DESCRIPTION
**Pull request by ElijahCapps**
## Summary of changes

I've added a picture of myself (Elijah Capps) to the website as well as edited the People.yml.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Checklist for Reviewers

Reviewers should use [this link](/manual/guides/pull_requests.md) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -169,6 +169,18 @@
...
+  dates: 2024-Present
+  role: Undergraduate Research Assistant
+  description: 'Eli is an undergraduate in NPRE interested in modeling nuclear energy systems.'
+  social:
+    - url: mailto:ecapps2@illinois.edu
+      icon: envelope-o
+      name: Email
+    - url: https://github.com/AbsoluteSlop
+      icon: github-alt
+      name: Github
```
Comment (yardasol) [inline _data/people.yml:183]: Is this your github account?

**Review by yardasol — COMMENTED** [review]
Hi Eli,
Overall this looks good. I only have one comment I'd like addressed before I approve.

**Comment by ElijahCapps** [comment]
Oops! I forgot that I pushed that before I changed my handle. Sorry about that. That is no longer my GitHub.
________________________________

**Comment by yardasol** [comment]
Great work @ElijahCapps

**Pull request merged**